javascript/javascript2/week1/Week1HW.js

```diff
@@ -0,0 +1,89 @@
+"use strict";
+
+//Find the shortest word
+
+const danishWords = ["bil", "plante", "kaffe", "bog", "ø", "planetarium"];
+const shortestWord = getShortestWord(danishWords); // returns 'ø'
+function getShortestWord(words) {
+  let newWord = danishWords[0];
+  for (let i = 0; i < danishWords.length; i++) {
+    if (danishWords[i].length <= newWord.length) {
+      newWord = danishWords[i];
+      danishWords[i] = danishWords[0];
+      danishWords[0] = newWord;
+    }
+  }
+  return danishWords[0];
+}
+document.getElementById("ShortestWord").innerHTML =
+  "shortest word in array : " + shortestWord;
+console.log(
+  'Shortest word in array ["bil", "plante", "kaffe", "bog", "ø", "planetarium"] is : ' +
+    shortestWord
+);
+
+//Find and count the Danish letters
+const danishString2 = "Blå grød med røde bær";
+let danishLetters = danishLettersCounter(danishString2);
+function danishLettersCounter(danString) {
+  const stringArray = danString.split(" ");
+  // console.log(stringArray);
+  let total = 0;
+  let åCount = 0;
+  let æCount = 0;
+  let øCount = 0;
+  for (let i = 0; i < stringArray.length; i++) {
+    if (stringArray[i].includes("å")) {
+      åCount++;
+      total++;
+    }
+    if (stringArray[i].includes("ø")) {
+      øCount++;
+      total++;
+    }
+    if (stringArray[i].includes("æ")) {
+      æCount++;
+      total++;
+    }
+  }
+
+  return (
+    "Toral Danish letter: " +
+    total +
+    "<br> No of æ: " +
+    æCount +
+    "<br> No of å: " +
+    åCount +
+    "<br> No of ø: " +
+    øCount
+  );
+}
+document.getElementById("danishLetters").innerHTML = danishLetters;
+console.log(danishLetters);
```

Comment:
you function does not return `{total: 4, æ: 1, ø: 2, å: 1}` but a string. Also if a letter has the zero count, you list it, where the solution should not be listing letters with zero count